vms/avm/wallet_service.go

```diff
@@ -82,11 +76,16 @@ func (w *WalletService) issue(txBytes []byte) (ids.ID, error) {
 	}
 
 	if w.pendingTxs.Len() == 0 {
-		_, err := w.vm.IssueTx(txBytes)
-		if err != nil {
+		if err := w.vm.mempool.Add(tx); err != nil {
```

Comment:
This is again a small behavior change - same rationale as above as to why I think it is fine.